sources/models/src/aws-ecs-1/override-defaults.toml

```diff
@@ -5,7 +5,7 @@ template-path = "/usr/share/templates/containerd-config-toml_aws-ecs-1"
 # ECS
 [services.ecs]
 restart-commands = ["/usr/bin/ecs-settings-applier", "/bin/systemctl try-reload-or-restart ecs.service"]
-configuration-files = ["ecs-config"]
+configuration-files = ["ecs-config", "proxy-env"]
```

Comment:
I'm not sure I understand tying the proxy-env configuration file to the ecs and kubernetes services.  It's more of a system-wide configuration file.  Can we have a new service defined in defaults.toml instead?  (That would handle aws-dev as well, which I'm not sure is handled now?)